flow/packet-headers/icmp.yaml

```diff
@@ -39,9 +33,14 @@ components:
             description: >-
               code of ICMP with next fields
             format: integer
-            length: 8
+            length: 1
             default: 0
             features: [count, metric_group]
+
+    Flow.ICMP.NextFields:
+      description: Next Fields for ICMP that contains ID, Seq Number and Next Fields
```

Comment:
Identifier and sequence number are 16bits each which makes up the rest of header.
The remainder of the packet is variable data up to a max of 576 bytes which can be filled in using the custom packet header which means next fields should not be present.